src/index.js

```diff
@@ -165,3 +165,7 @@ i18next.on('initialized', () => {
   showSlowLoadingWindow();
 });
 
+const origin = window.location.origin;
+const pathname = window.location.pathname;
+document.querySelector('#ngm-home-link').href = origin.includes('review') ? `${origin}${pathname}` : origin;
```

Comment:
can it be simplified to:
```suggestion
document.querySelector('#ngm-home-link').href = `${origin}${pathname}`;
```
If it's not `review` `pathname` is `/` and the url is something like `https://dev.swissgeol.ch/` (which is correct)

Reply:
updated